task/cache.py

```diff
@@ -48,10 +50,10 @@ def __init__(self, directory, lag=None):
         self.lag = lag
 
         # The mark tells us that stuff before this time is not "current"
-        self.marked = 0
+        self.marked: float = 0
```

Comment:
ditto comment about the variable annotation being on the class

Reply:
Agreed, fixed.

Reply:
This isn't fixed.